packages/api-reference/src/components/Content/Introduction/Introduction.vue

```diff
@@ -23,33 +23,48 @@ const props = defineProps<{
   parsedSpec: Spec
 }>()
 
-const slugger = new GithubSlugger()
-
-const specVersion = computed(() => {
-  return props.parsedSpec.openapi ?? props.parsedSpec.swagger ?? ''
+/**
+ * Get the OpenAPI/Swagger specification version from the API definition.
+ */
+const oasVersion = computed(() => {
+  return props.parsedSpec?.openapi ?? props.parsedSpec?.swagger ?? ''
 })
```

Comment:
```suggestion
const oasVersion = computed(() => props.parsedSpec?.openapi ?? props.parsedSpec?.swagger ?? '')
```
the beauty of arrow functions is no need to write return!